state_machine/applications/flight/Tasks/telemetry.py

```diff
@@ -20,15 +19,17 @@ async def main_task(self):
         try:
             self.write_beacon()
         except Exception:
-            files.mkdirp('/sd/logs/beacon/')
+            files.mkdirp('/sd/logs/telemetry/')
             self.write_beacon()
 
     def write_beacon(self):
-        currTime = time.time()
-        TIMEINTERVAL = 1000
-        log_directory = "/sd/logs/"
-        current_file = f"{log_directory}beacon/{int(currTime//TIMEINTERVAL)}.txt"
+        t = cubesat.rtc.datetime
+        hour_stamp = f'{t.tm_year}.{t.tm_mon}.{t.tm_mday}.{t.tm_hour}'
+        current_file = f"/sd/logs/telemetry/{hour_stamp}.txt"
+        time_packet = logs.time_packet(t)
```

Comment:
Make a `logs.telemetry_packet()` function that takes care of concatenating the three packets`

Reply:
I made a function that concatenates the bytearrays of each packet and then returns that bytearray